src/plugins/PluginFactory.cpp

```diff
@@ -175,13 +175,17 @@ long PluginFactory<PluginBase>::enablePlugins(std::list<PluginBase*>& _plugins,
 		}
 		Log::global_log->info() << "Enabling plugin: " << pluginname << std::endl;
 
-		PluginBase* plugin = this->create(pluginname);
-		if (plugin == nullptr) {
-			Log::global_log->warning() << "Could not create plugin using factory: " << pluginname << std::endl;
+		// Allowing an alias for a plugin. Not sure why this exists...
+		if (pluginname == "DomainProfiles") {
+			pluginname = "DensityProfileWriter";
+			Log::global_log->warning() << "DomainProfiles doesn't exist but is mapped to DensityProfileWriter!\n";
 		}
```

Comment:
I'm not sure how this plugin works. When I start any example that contains it from master they also don't manage to load it. Looking at the code it seems to be very old and most likely unused for decades.